{
    "CVE_data_meta": {
        "ASSIGNER": "cve@mitre.org",
        "ID": "CVE-2010-2913",
        "STATE": "PUBLIC",
        "UPDATED": "2020-06-15"
    },
    "cna-container": {
        "affected": {
            "vendors": [
                {
                    "products": [
                        {
                            "versions": [
                                {
                                    "version_value": "n/a"
                                }
                            ]
                        }
                    ],
                    "vendor_name": "n/a"
                }
            ]
        },
        "descriptions": [
            {
                "lang": "eng",
                "value": "The Citibank Citi Mobile app before 2.0.3 for iOS stores account data in a file, which allows local users to obtain sensitive information via vectors involving (1) the mobile device or (2) a synchronized computer."
            }
        ],
        "problemtypes": {
            "descriptions": [
                [
                    {
                        "description": "n/a",
                        "lang": "eng",
                        "type": "text"
                    }
                ]
            ]
        },
        "references": [
            {
                "name": "http://news.cnet.com/8301-27080_3-20011664-245.html",
                "refsource": "MISC",
                "url": "http://news.cnet.com/8301-27080_3-20011664-245.html"
            },
            {
                "name": "1024249",
                "refsource": "SECTRACK",
                "url": "http://securitytracker.com/id?1024249"
            },
            {
                "name": "http://itunes.apple.com/us/app/citi-mobile-sm/id301724680",
                "refsource": "MISC",
                "url": "http://itunes.apple.com/us/app/citi-mobile-sm/id301724680"
            },
            {
                "name": "citimobile-data-information-disclosure(60855)",
                "refsource": "XF",
                "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/60855"
            }
        ]
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "5.0"
}